WordPress/gutenberg
# Env: Escape Double Quotes in WordPress Setup Commands

## What?
This pull request fixes shell command errors caused by the setup commands in `wp-env start`.

## Why?
The lack of escaping for the double quotes in the WordPress setup commands is leading to errors. Locally I am using `Multipass` to run Docker commands in a VM using their alias feature. As far as I can tell, the abstraction between `zsh` and the actual execution of `docker-compose` is causing the double quotes to break.

## How?
We're going to escape all of the double quotes in the WordPress setup commands.

## Testing Instructions
1. Ensure that `wp-env start` works in your current environment. This should be a good indicator of backward compatibility.
2. Ensure that CI passes. This one is a given, but, it's another indicator that the change is backward compatible.
